athena-elasticsearch/src/main/java/com/amazonaws/athena/connectors/elasticsearch/ElasticsearchMetadataHandler.java

```diff
@@ -185,8 +185,13 @@ public ListTablesResponse doListTables(BlockAllocator allocator, ListTablesReque
                 .stream()
                 .filter(index -> !index.startsWith("."));
 
+        // get all data streams from ES, 1 data stream can contains multiple indices which start with ".ds-xxxxxxxxxx"
+        Stream<String> dataStreams = client.indices().getDataStream(new GetDataStreamRequest("*"), RequestOptions.DEFAULT).getDataStreams()
+                .stream()
+                .map(dataStream -> dataStream.getName());
+
         //combine two different data sources and create tables
-        List<TableName> tableNames = Stream.concat(indicesStream, getDataStreamNames(client))
```

Comment:
when we are rebasing from the master, we observed that this file alone is not getting reflected with the latest master changes and in our PR this is the only additional file coming. Please suggest if we are doing anything wrong.

Another solution could be updating this file from the latest awslabs master and pushing the changes. But it will reflect as a changed file from this PR.

Reply:
This still has not been fixed after the latest pushes I saw. Please address this as it is a merge blocker @Trianz-Akshay

Reply:
Hi @Trianz-Akshay - for some reason, GitHub did not show me your comments when I was doing my review earlier, and I'm only now seeing your two messages here; I apologize as it makes my comment look rude.

The easiest way to fix this without getting into more complicated Git commands is going to be to just copy the file from master paste it onto your review branch. Then, squash all of your commits. That way, the diff will be empty for this file.

Reply:
thanks, resolved this.